src/main/java/ru/ayaz/Comment.java

```diff
@@ -0,0 +1,23 @@
+package ru.ayaz;
+
+import java.util.Calendar;
+import java.util.Date;
+
+public class Comment {
```

Comment:
Yes, I know) I had to override equals for UserMessage class in Chat project.
I hesitated a little bit because i had mocked object. So i need to configure mock behavior and rewrite and test equals method.